sync/class.jetpack-sync-defaults.php

```diff
@@ -191,6 +191,7 @@ public static function get_constants_whitelist() {
 		'hosting_provider'                 => array( 'Jetpack_Sync_Functions', 'get_hosting_provider' ),
 		'locale'                           => 'get_locale',
 		'site_icon_url'                    => array( 'Jetpack_Sync_Functions', 'site_icon_url' ),
+		'get_editable_roles'               => 'get_editable_roles',
```

Comment:
I think you might need to add a test for this. And also sanitize stuff on the .com side of things.

Reply:
👍 on it